model/channel.go

```diff
@@ -73,6 +73,11 @@ type ChannelForExport struct {
 	SchemeName *string
 }
 
+type DirectChannelForExport struct {
+	Channel
+	Members *[]string
```

Comment:
Making `Members` a pointer here seems to be redundant (arrays are already pointers). Can we simplify?

Reply:
I guess the previous contributor tried to keep it consistent with what's present in most of the models. In `import_types.go` you'll see that most of the slices are pointers. It's a valid review point but I'd tend more to keep it consistent by either changing all slice pointers (as in *[]string) to plain []string across the board (fixing the code that will break).
All that to say that I agree with you and that we should try to avoid that in the way forward.